locales/en/form.json

```diff
@@ -81,6 +82,6 @@
     "uploading": "Uploading..."
   },
   "use_in_observation": "Use Selected",
-  "wkt": "WKT",
+  "wkt": "Search Point",
```

Comment:
please check if `wkt` key is used anywhere
also if possible please add new key with `search_point` insead of using `wkt` (of course remove `wkt` key if not in use)